src/cli/cli.cc

```diff
@@ -1198,6 +1198,16 @@ int main (const int argc, const char* argv[]) {
         }
       }
 
+      if (settings["android_allow_cleartext"].size() == 0)
+      {
```

Comment:
We prefer left curly braces to be in the same line as the right paren

Reply:
👍